index.ts

```diff
@@ -41,22 +51,53 @@ const CUSTOMERS = [
   },
 ];
 
+
+
 // create a function that receives a name of a customer, and returns the balance of that customer.
 // if type of this function is not infered, type it.
-
 // EXTRA: optimize this fucnction to decrease the time complexity.
 // EXTRA 2 : optimize this function to be able to memorize the results of previous calls.
-
 // i: getBalance('John')
 // o: 125
+type Customer = {
+  customer: string;
+  balance: number;
+  id: string;
+}
```

Comment:
there is no need to add another type that could be infered from previous definitions, learn about it